### PR TITLE
Remove references to TFE_TensorHandleIsConcrete

### PR DESCRIPTION
**Pull request by jaingaurav**

### CODE REVIEW

File: Sources/TensorFlow/Core/Tensor.swift [hunk 1]
```diff
@@ -158,7 +158,6 @@ public extension Tensor {
     @inlinable
     var array: ShapedArray<Scalar> {
         debugLog("Returning a host copy of array.")
-        internalConsistencyCheck(handle.isConcrete)
 	    return handle.makeHostCopy()
```
Comment (eaplatanios) [inline Sources/TensorFlow/Core/Tensor.swift]:
```suggestion
      return handle.makeHostCopy()
```

Comment (jaingaurav) [reply Sources/TensorFlow/Core/Tensor.swift]: Thanks, it should be fixed now. I believe there was a mixture of some tabs in the file which caused the indentation issues.

Comment (eaplatanios) [reply Sources/TensorFlow/Core/Tensor.swift]: Thanks @jaingaurav! Yes it looked like some tabs were being used.

File: Sources/TensorFlow/Core/Tensor.swift [hunk 2]
```diff
@@ -224,8 +223,8 @@ public extension Tensor {
             provided.
             """)
         self = scalars.withUnsafeBufferPointer { bufferPointer in
-	        Tensor(shape: shape, scalars: bufferPointer)
-	    }
+          Tensor(shape: shape, scalars: bufferPointer)
```
Comment (eaplatanios) [inline Sources/TensorFlow/Core/Tensor.swift]:
```suggestion
            Tensor(shape: shape, scalars: bufferPointer)
```

**Comment by bgogul** [comment]
Thanks, @jaingaurav. It looks like there are a bunch of places where the function is still being used: 

https://source.cloud.google.com/results/invocations/22c80a3d-3b56-4666-a31d-935c77b552fb/targets/s4tf%2Fapis%2Fubuntu%2Fpresubmit/log

If you want to test this, You will need to get the latest swift toolchain from https://github.com/tensorflow/swift/blob/master/Installation.md

If it makes it easier for you, I can go ahead and fix this. Can you allow me (bgogul) to push changes to your branch?

**Pull request merged**